src/transaction/log_manager.c

```diff
@@ -4450,6 +4463,38 @@ log_append_donetime_internal (THREAD_ENTRY * thread_p, LOG_TDES * tdes, LOG_LSA
   LSA_COPY (eot_lsa, &lsa);
 }
 
+static void
+log_append_group_commit (THREAD_ENTRY * thread_p, LOG_TDES * tdes, INT64 stream_pos, const tx_group & group,
+			 LOG_LSA * commit_lsa)
+{
+  LOG_PRIOR_NODE *node;
+  LOG_LSA lsa;
+
+  commit_lsa->pageid = NULL_PAGEID;
+  commit_lsa->offset = NULL_OFFSET;
+
+  // *INDENT-OFF*
+  std::vector<gc_tran_info> v;
+  for (const auto & ti : group.get_container ())
+    {
+      v.push_back ({ti.m_tran_index, ti.m_tran_state});
+    }
+  // *INDENT-ON*
+
+  node =
+    prior_lsa_alloc_and_copy_data (thread_p, LOG_GROUP_COMMIT, RV_NOT_DEFINED, NULL, 0, NULL,
+				   v.size () * sizeof (gc_tran_info), (const char *) v.data ());
+
+  LOG_REC_GROUP_COMMIT *gc = (LOG_REC_GROUP_COMMIT *) node->data_header;
+  gc->at_time = time (NULL);
```

Comment:
I think we need to consider more precise time (millisecond).
We may enhance it later.